### DISCUSSION TITLE
Certificate generator problem

### DISCUSSION THREAD

**Discussion by SYRTI** [wazuh/wazuh-docker#727] [Q&A] [answered]
Hi team, 

I'm have a problem when try to create certificates with
docker-compose -f generate-indexer-certs.yml run --rm generator

I have the following error.

Status: Downloaded newer image for wazuh/wazuh-certs-generator:0.0.1
Creating single-node_generator_run ... done
Cert tool does not exist in any bucket
ERROR: certificates were not created
ERROR: 1

with this error can't docker-compose up because the admin.pem isn't created

Creating single-node_wazuh.indexer_1 ... error
Creating single-node_wazuh.manager_1 ... 

ERROR: for single-node_wazuh.indexer_1  Cannot start service wazuh.indexer: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/home/wazuh/wazuh/wazuh-docker/single-node/config/wazuh_indexer_ssl_certs/admin.pem" to rootfs at "/usr/share/wazuh-indexer/config/certs/admin.pem": mount /home/wazuh/wazuh/wazuh-docker/single-node/config/wazuh_indexer_ssl_certs/admin.pem:/usr/share/wazuh-indexer/config/certs/admin.pem (via /proc/self/fd/6), flags: 0x5000: not a directory: unknown: Are you trying to mount a dCreating single-node_wazuh.manager_1 ... done

ERROR: for wazuh.indexer  Cannot start service wazuh.indexer: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/home/wazuh/wazuh/wazuh-docker/single-node/config/wazuh_indexer_ssl_certs/admin.pem" to rootfs at "/usr/share/wazuh-indexer/config/certs/admin.pem": mount /home/wazuh/wazuh/wazuh-docker/single-node/config/wazuh_indexer_ssl_certs/admin.pem:/usr/share/wazuh-indexer/config/certs/admin.pem (via /proc/self/fd/6), flags: 0x5000: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type
ERROR: Encountered errors while bringing up the project.
root@wazuh-virtual-machine:/home/wazuh/wazuh/wazuh-docker/single-node# 

can you help me please? I'm doing something wrong?

Best Regards

**Comment by rauldpm** [comment, marked as answer]
Since there is an open issue from the same user for this with more information where another user has already participated, this discussion will be continued in the issue: https://github.com/wazuh/wazuh-docker/issues/733.

In the meantime, this discussion will be in a `Locked` state to avoid duplicates, please, for any new information or participation use the named issue.
